### ISSUE TITLE
Import date as well as bill date

### ISSUE THREAD

**Issue by Phyks** [spiral-project/ihatemoney#325]
Hi,

There is currently a single date field on bills, reflecting the date of the bill. 

It is usually helpful to have the date of addition as well. This is especially useful when adding long past bills so that the date reflects the real date of the bills, but to allow a way to sort and see last additions. 

What do you think about adding a new date field on the bill to store date of last update? I can write a PR for this if you are interested :)

Thanks,

**Comment by almet** [comment]
Hi @Phyks, if you have a use case where this new date field would be useful, go ahead!

**Comment by Phyks** [comment]
Hi,

I made a basic PR to propose it and discuss it a bit. My typical use case is that I'd like the date of the bill to be the real date, that is the date I paid on (for easy matching with my bank account for instance).

We tend to enter expenses sometimes a long time after we actually paid (typically adding a bunch of utilities bills at the same time, for the past months), and then it is super difficult for the others to see that I indeed added something. Being able to filter on `imported_date` would put these bills at the top, while keeping the correct date for the expense.
